test/e2e/azure_machinepool_drain.go

```diff
@@ -275,14 +283,17 @@ func deployHttpService(ctx context.Context, clientset *kubernetes.Clientset, isW
 		Eventually(func() error {
 			version, err := node.GetWindowsVersion(ctx, clientset)
 			windowsVersion = version
+			if err != nil {
+				LogWarning(err.Error())
+			}
 			return err
 		}, 300*time.Second, 5*time.Second).Should(Succeed())
```

Comment:
same here about using a const

Reply:
this is only used once (and this one is inherited, should we standardize it to the newly added "try every 3 seconds up to 30 seconds" config?)

Reply:
In fact the underlying node list call in `node.GetWindowsVersion` is now implementing retries. This is the right place for this, so I'll get rid of the `Eventually` wrapping here altogether.